content/en/docs/ops/configuration/mesh/app-health-check/index.md

```diff
@@ -32,6 +32,68 @@ TCP probe checks need special handling, because Istio redirects all incoming tra
 
 Istio solves both these problems by rewriting the application `PodSpec` readiness/liveness probe,
 so that the probe request is sent to the [sidecar agent](/docs/reference/commands/pilot-agent/).
+
```

Comment:
If you add a section name here, the snippets will use the section name instead of just snip_x. Might make the test easier to read and maybe help the doc reader.

Reply:
agree

Reply:
agree